spec/requests/api/v1/user_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UserController, type: :request do
+  describe 'GET /api/v1/user' do
+    let!(:user) { create(:user) }
+    let(:object) { JSON.parse(response.body)}
+
+    it 'Should return all user' do
+      get '/api/v1/user', headers: {
+        email: user.email, password: user.password
+      }
+
+      expected_response = eval(file_fixture('users.txt').read)
+    end
+
+    it 'Should return one user' do
+      get '/api/v1/user/:id', headers: {
+        email: user.email, password: user.password
+      }
+
+      expected_response = eval(file_fixture('user.txt').read)
```

Comment:
You are actually not testing anything here. 
You need to add something like this after this line.
``` 
expect(object['data'].first).to eq expected_response
``` 
this just reads the user.txt file and sets expected_response equal to it :)
```
expected_response = eval(file_fixture('user.txt').read)
```

Just a heads up.